Core/include/Acts/Propagator/Propagator.ipp

```diff
@@ -258,11 +252,6 @@ auto Acts::Propagator<S, N>::makeState(
   StateType state{eOptions, m_stepper.makeState(eOptions.stepping),
                   m_navigator.makeState(eOptions.navigation)};
 
```

Comment:
This can't happen anymore? I believe the idea was mainly to give a better error message.